apps/time/src/widgets/sugang/ui/SugangLayout.tsx

```diff
@@ -0,0 +1,83 @@
+'use client';
+
+import { Dispatch, useState } from 'react';
+
+import { KguLogo } from '@clab-platforms/icon';
+import SvgMenu from '@clab-platforms/icon/src/outline/react/Menu';
+
+import { SUGANG_PAGES } from '@/widgets/sugang/model';
+import { SugangTableView } from '@/widgets/sugang/ui';
+import { SetStateAction } from 'jotai';
+
+export default function SugangLayout() {
+  const [opened, setOpened] = useState(false);
+
+  return (
+    <div className="flex size-full">
+      <SideBar opened={opened} />
+      <div className="flex size-full flex-col">
+        <Header setOpened={setOpened} />
+        <div className="mb-2 flex size-full flex-col gap-y-2 p-2 px-4">
+          <NavigatorBtn />
+          <SugangTableView tableName="wishList" />
+          {/*현재는 tableName이 고정되어 있지만, 나중에는 NavigatorBtn을 통해 누른 탭에 따라 Name을 변하게 할 예정입니다.*/}
+          <SugangTableView tableName="registrationList" />
+        </div>
+      </div>
+    </div>
+  );
+}
+
+function SideBar({ opened }: { opened: boolean }) {
+  return (
+    <div
+      className={`h-full w-44 flex-col bg-yellow-500 ${opened ? 'flex' : 'hidden'}`}
```

Comment:
```suggestion
     className={cn('h-full w-44 flex-col bg-yellow-500', opened ? 'flex' : 'hidden')}
```

`utils`의 `cn`을 사용하여 조건부 스타일링을 적용할 수 있어요.

Reply:
써본다고 하고 사용하지 않았네요.. 감사합니다!